requirements.txt

```diff
@@ -12,4 +12,5 @@ numpy
 pyyaml
 datasets
 psutil
-setuptools
\ No newline at end of file
+setuptools
+Ninja
```

Comment:
Do we need this? I think it should be removed. If this is required for tests to pass, you can add it to `extras["test"] in `setup.py`.

Reply:
Deleted.

Reply:
Deleted.